docs/concepts/glossary.md

```diff
@@ -0,0 +1,168 @@
+data set is divided into partitions by assigning each [record](./glossary#Record) to one or more
+partitions. Partitions of data streams or data sets are consumed by [Tasks](./glossary#task) during
+runtime. A transformation which changes the way a data stream or data set is partitioned is often
+called repartitioning.
+
+#### Physical Graph
+
+A physical graph is the result of translating a [Logical Graph](./glossary#logical-graph) for
+execution in a distributed runtime. The nodes are [Tasks](./glossary#task) and the edges indicate
+input/output-relationships or [partitions](./glossary#partition) of data streams or data sets.
+
+#### Record
+
+Records are the constituent elements of a data set or data stream.
+[Operators](./glossary#operator) and [Functions](./glossary#Function) receive records as input
+and emit records as output.
+
+#### Flink Session Cluster
+
+A long-running [Flink Cluster](./glossary#flink-cluster) which accepts multiple
+[Flink Jobs](./glossary#flink-job) for execution. The lifetime of this Flink Cluster is not bound
+to the lifetime of any Flink Job. Formerly, a Flink Session Cluster was also known as a Flink Cluster in
+*session mode*. Compare to [Flink Application Cluster](./glossary#flink-application-cluster).
+
+#### State Backend
+
+For stream processing programs, the State Backend determines how state is stored on each Taskmanager
+(Java Heap of Taskmanager or (embedded) RocksDB) as well as where it is written upon a checkpoint
+(Java Heap of [Flink JobManager Process](./glossary#flink-jobmanager-process) or Filesystem).
+
+#### Sub-Task
+
+A Sub-Task is an instance of [Task](./glossary#task) responsible for processing a
+[partition](./glossary#partition) of the data stream. The term "Sub-Task" emphasizes that there are
+multiple parallel Tasks for the same [Operator](./glossary#operator).
+
+#### Task
+
+Node of a [Physical Graph](./glossary#physical-graph). A task is the basic unit of work, which is
+executed by Flink's runtime. Tasks encapsulate exactly one parallel instance of an
+[Operator](./glossary#operator) or [Operator Chain](./glossary#operator-chain).
+
+#### Flink TaskManager
+
+TaskManagers are the worker processes of a [Flink Cluster](./glossary#flink-cluster).
+[Tasks](./glossary#task) are scheduled to TaskManagers for execution. They communicate with each
+other to exchange data between subsequent Tasks.
+
+#### Transformation
+
+A Transformation is applied on one or more data streams or data sets and results in one or more
+output data streams or data sets. A transformation might change a data stream or data set on a
+per-record basis, but but might also only change its partitioning or perform an aggregation. While
+[Operators](./glossary#operator) and [Functions](./glossary#function)) are "physical" parts Flink's
+API, Transformations are only an API concept. Specifically, most - but not all -  transformations
```

Comment:
there is one space too many after the last "-"

Reply:
Done.